external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaSource.scala

```diff
@@ -188,6 +192,8 @@ private[kafka010] class KafkaSource(
         // checking if we need to skip batch based on minOffsetPerTrigger criteria
         val skipBatch = delayBatch(
           limit.minRows, latest, currentOffsets.get, limit.maxTriggerDelayMs)
+
+        println(s"skipBatch: $skipBatch limit.minRows: ${limit.minRows} latest: $latest limit.maxTriggerDelayMs: ${limit.maxTriggerDelayMs}")
```

Comment:
Please change this to logDebug, or delete this if you think it's no longer needed. It seems to me worth to leave as debug or trace log message for further debugging.

Reply:
Btw I'd like to see this be added in micro batch stream as well if we want to retain this here.

Reply:
I will remove